src/XMakeBuildEngine/Resources/Strings.resx

```diff
@@ -614,6 +614,9 @@
     <comment>{StrBegin="MSB4035: "}UE: This message is shown when a user leaves off a required attribute from a project element
     e.g. &lt;UsingTask AssemblyName="foo"&gt; -- this is missing the "TaskName" attribute.</comment>
   </data>
+  <data name="IncludeRemoveOrUpdate" UESanitized="false" Visibility="Public">
+    <value>MSB4232: Static items (items not under targets) must have one of the following operations: Include, Update, or Remove.</value>
```

Comment:
Have we used "static items" in this sense before in user-facing messages? I think I'd rephrase to just `Items that are outside Target elements must . . .`, which matches https://msdn.microsoft.com/en-us/library/ms171453.aspx#Anchor_1